Hackaton 2024: resolved some doc issues (#2574)
openturns/openturns: python/doc/theory/probabilistic_modeling/estimate_spectral_density_function.rst

.. _estimate_spectral_density_function:

Estimation of a spectral density function
-----------------------------------------

Let :math:`X: \Omega \times \cD \rightarrow \Rset^{\inputDim}` be a multivariate
stationary normal process of dimension :math:`d`. We only treat here
the case where the domain is of dimension 1: :math:`\cD \in \Rset`.
If the process is continuous, then :math:`\cD=\Rset`. In the discrete
case, :math:`\cD` is a lattice.
:math:`X` is supposed to be a second order process with zero mean and
we suppose that its spectral density function
:math:`S : \Rset \rightarrow \mathcal{H}^+(\inputDim)` defined in
:eq:`specdensFunc` exists.
:math:`\mathcal{H}^+(\inputDim) \in \mathcal{M}_{\inputDim}(\Cset)` is the set of

> :math:`\inputDim`-dimensional positive definite hermitian matrices.

Reviewer: ```suggestion
:math:`\inputDim`-dimensional positive definite Hermitian matrices.
```

> This objective of this use case is to estimate the spectral density

Reviewer: ```suggestion
Our goal is to estimate the spectral density
```
